.github/actions/setup-local-fal/action.yml

```diff
@@ -51,8 +51,14 @@ runs:
           exit 2
         fi
 
-        echo "${package}==${version}"
-        pip install ${package}==${version}
+        if [[ '${{ inputs.adapter }}' == 'fal' ]]
+        then
+          pip install -e ./adapter
```

Comment:
```suggestion
          pip install -e './adapter[postgres]'
```

Reply:
Also, should we install `dbt-fal` instead of the local version? 

PRO: not coupling them
CON: test latest changes together

Reply:
I think for now it's a good idea to keep them coupled. Maybe until 1.4?